cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/Location.java

```diff
@@ -486,6 +488,25 @@ public void setOfficeId(String officeId) {
         this.officeId = officeId;
     }
 
+    public Location active(boolean active) {
+        this.active = active;
+        return this;
+    }
+
+    /**
+     * Get officeId
+     *
+     * @return officeId
+     **/
+
+    public boolean isActive() {
+        return active;
+    }
+
+    public void setActive(boolean active) {
```

Comment:
This one is just pure curiousity. Was the choice to use mutable objects an active one or did was this just the default?

honestly a little late to change, but inquiring minds want to know. I made a fairly forceful decision on the server side just to make sure there was one less possible side affect to worry about.... and make sure things were always in a reasonable state.

Reply:
Yes, the mutability comes from the swagger code generator. The setActive() is for Java Bean standards, while the active() is for fluent API.

The swagger code generator does provide the ability to edit the template so that I could remove the mutability, but like Peter said, we will need to be able to set these variables for POST. Unfortunately the code generator does not provide options for generating a builder for the model objects.

Reply:
well, that's a "good" reason. Apparently just isn't something we can avoid.